src/sockets/tests/arestelo.test.ts

```diff
@@ -0,0 +1,54 @@
+import { aresetelo } from "../commands/admin/aresetelo";
+import User from "../../models/user";
+jest.mock('../../models/user');
+jest.mock('../../models/modLog');
+
+describe('aresetelo Commands', () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('calls aresetelo', async () => {
+    const args = ['/aresetelo'];
+
+    const mockSocket = {
+      emit: jest.fn(),
+      request: {
+        user: {
+          username: 'pronub',
+        },
+      },
+    };
+    const mockUsers = [
+      {
+        _id: 'user1',
+        username: 'user1',
+        currentRanking: 'oldRanking1',
+      },
+      {
+        _id: 'user2',
+        username: 'user2',
+        currentRanking: 'oldRanking2',
+      },
+    ];
+
+    const userFindMock = User.find as jest.MockedFunction<typeof User.find>;
+    userFindMock.mockResolvedValue(mockUsers);
+
+    // @ts-ignore
+    await aresetelo.run(args, mockSocket);
```

Comment:
Let's simplify this by not calling directly into the run() method. Instead, you can simply give in mock User objects and directly call `resetElosOfUsers(users);` as this is really the main domain logic we want to test here!

If you'd like, you can add a larger test that does this as well. Let's sit down in a meeting sometime to work it out. Would be good also assert that season number increments which we can only do by doing this.